.gitignore

```diff
@@ -12,3 +12,4 @@ test.dat
 test.npy
 test.out
 tree-sitter-rst/
+.idea/
```

Comment:
You want to put that in your global `.gitignore` https://gist.github.com/subfuzion/db7f57fff2fb6998a16c, not on each project you contribute to.

Reply:
Ah, that's very useful!